src/Promenade.Web/Views/Podcasts/Episode.cshtml

```diff
@@ -2,12 +2,29 @@
 
 <div class="row">
     <div class="col-12">
-        <h1 class="mb-2">@Model.PodcastItem.Title</h1>
-        <div class="mb-4">
-            <em>
-                <a href="@Url.Action(nameof(PodcastsController.Podcast), new { stub = Model.Podcast.Stub})">@Localizer[Promenade.PodcastTitle, Model.Podcast.Title]</a>.
-                @Localizer[Promenade.PodcastEpisodePublished, Model.PodcastItem.Episode, Model.PodcastItem.PublishDate.Value.ToLongDateString()]
-            </em>
+        <div class="row">
+            <div class="col-12 col-lg-6 col-xl-7">
```

Comment:
One small design change, this can be changed to `col-12 col-lg` and the columns on Line#15 can be changed to `col-12 col-lg-auto`. This will create a column that auto-sizes itself to the View show notes button and allow the title column to fill in the rest of the space instead of trying to shift the column width at every view size.

Reply:
Resolved in commit eb3b15b